server/src/unifycr_request_manager.c

```diff
@@ -359,148 +366,156 @@ int rm_receive_remote_message(int app_id,\
+
+    }
+
+    return ULFS_SUCCESS;
 
 }
 
-int rm_init(int size) {
-	/*
-	req_dels_stat.stat =\
-			(delegator_stat_t *)malloc(sizeof(delegator_stat_t));
-	if (!req_dels_stat.stat)
-		return ULFS_ERROR_RM_INIT;
-		*/
-	return ULFS_SUCCESS;
+int rm_init(int size)
+{
+    /*
+    req_dels_stat.stat =
+            (delegator_stat_t *)malloc(sizeof(delegator_stat_t));
+    if (!req_dels_stat.stat)
+        return ULFS_ERROR_RM_INIT;
+        */
+    return ULFS_SUCCESS;
 }
 
-int compare_delegators(void *a, void *b) {
-	if (((send_msg_t *)a)->dest_delegator_rank\
-			- ((send_msg_t *)b)->dest_delegator_rank > 0)
-		return 1;
-	if (((send_msg_t *)a)->dest_delegator_rank\
-			- ((send_msg_t *)b)->dest_delegator_rank < 0)
-		return -1;
+int compare_delegators(void *a, void *b)
+{
+    if (((send_msg_t *)a)->dest_delegator_rank
+        - ((send_msg_t *)b)->dest_delegator_rank > 0) {
+        return 1;
+    }
+    if (((send_msg_t *)a)->dest_delegator_rank
+        - ((send_msg_t *)b)->dest_delegator_rank < 0) {
+        return -1;
+    }
 
-	return 0;
+    return 0;
 }
 
-void print_send_msgs(send_msg_t *send_metas,\
-		long msg_cnt, int dbg_rank) {
-	long i;
-	for (i = 0; i < msg_cnt; i++) {
-		LOG(LOG_DBG, "print_send_msgs:dbg_rank:%d,\
-				 src_offset:%ld, msg_cnt:%ld\n", \
-				dbg_rank, send_metas[i].src_offset, msg_cnt);
-	}
+void print_send_msgs(send_msg_t *send_metas,
+                     long msg_cnt, int dbg_rank)
+{
+    long i;
+    for (i = 0; i < msg_cnt; i++) {
+        LOG(LOG_DBG, "print_send_msgs:dbg_rank:%d, \
+            src_offset:%ld, msg_cnt:%ld\n",
+            dbg_rank, send_metas[i].src_offset, msg_cnt);
+    }
 }
 
-void print_remote_del_reqs(int app_id, int cli_id, \
-		int dbg_rank, del_req_stat_t *del_req_stat) {
-	int i;
-	for (i = 0; i < del_req_stat->del_cnt; i++) {
-		LOG(LOG_DBG, "remote:dbg_rank:%d, remote_delegator:%d, req_cnt:%d===\n",\
-				dbg_rank, del_req_stat->req_stat->del_id,\
-					del_req_stat->req_stat->req_cnt);
-		fflush(stdout);
+void print_remote_del_reqs(int app_id, int cli_id,
+                           int dbg_rank, del_req_stat_t *del_req_stat)
+{
+    int i;
+    for (i = 0; i < del_req_stat->del_cnt; i++) {
+        LOG(LOG_DBG, "remote:dbg_rank:%d, remote_delegator:%d, req_cnt:%d===\n",
+            dbg_rank, del_req_stat->req_stat->del_id,
+            del_req_stat->req_stat->req_cnt);
+        fflush(stdout);
 
-	}
+    }
 
 }
 
-void print_recv_msg(int app_id,\
-		int cli_id, int dbg_rank, int thrd_id, shm_meta_t *msg) {
-	LOG(LOG_DBG, "recv_msg:dbg_rank:%d, app_id:%d, cli_id:%d, thrd_id:%d,\
-			  fid:%d, offset:%ld, len:%ld\n", \
-			  dbg_rank, app_id, cli_id,  thrd_id, msg->src_fid,\
-				  	  msg->offset, msg->length);
+void print_recv_msg(int app_id,
+                    int cli_id, int dbg_rank, int thrd_id, shm_meta_t *msg)
+{
+    LOG(LOG_DBG, "recv_msg:dbg_rank:%d, app_id:%d, cli_id:%d, thrd_id:%d, \
+        fid:%d, offset:%ld, len:%ld\n",
```

Comment:
Unrelated to your changes, but this quoted string has a formatting error. Should remove line continuation and use two quoted strings.